.github/prep-hotfix-sources.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash -x
+
+HOTFIX_TAG=$1
+REPO_PATH=$2
+SPEC_PATH=$3
```

Comment:
Is it allowed for any of these PATH variables have whitespace or special characters?

(Because if so you will need to quote any use of these variables, and any use of variables based off them.)